azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/header/InfoHeaderViewModel.kt

```diff
@@ -50,6 +50,20 @@ internal class InfoHeaderViewModel(
         }
     }
 
+   fun getFormattedElapsedDuration(): String {
+      val elapsedDuration = callDurationManager?.getElapsedDuration() ?: 0L
+      val seconds = (elapsedDuration / 1000) % 60
+      val minutes = (elapsedDuration / (1000 * 60)) % 60
+      val hours = (elapsedDuration / (1000 * 60 * 60)) % 24
+
+      return when {
+        hours > 0 -> String.format("%02d:%02d:%02d", hours, minutes, seconds)
```

Comment:
Hardcoded string, should be coming through the string resource system/be localizable.

Reply:
We should test on the device with localization different from time format in US

Reply:
Do I need to add strings by myself in localization?